torchtune/data/_converters.py

```diff
@@ -55,3 +55,59 @@ def sharegpt_to_llama2_messages(
         masked = (role != "assistant") and (not train_on_input)
         messages.append(Message(role=role, content=content, masked=masked))
     return messages
+
+
+def standard_chat_to_llama2_messages(
+    sample: Mapping[str, Any],
+    train_on_input: bool = False,
+) -> List[Message]:
+    """
+    Convert a chat sample adhering to the OpenAI API standard chat format to the Llama2 chat format.
```

Comment:
Do you know if there's a reference for this?

Reply:
Yes! We do at https://platform.openai.com/docs/guides/text-generation/chat-completions-api

I will add that to the code :)